src/neo/Ledger/Blockchain.state.cs

```diff
@@ -0,0 +1,12 @@
+using Akka.Actor;
+
+namespace Neo.Ledger
+{
+    public sealed partial class Blockchain : UntypedActor
+    {
+        public UInt256 GetLocalStateRoot(uint index)
+        {
+            return UInt256.Zero;
```

Comment:
It will be filled in other PR?

Reply:
Yes. My changes in consensus need interface in Blockchain. And blockchain changes need persistence changes merged. I have no choice.